Code/Tools/SceneAPI/SceneData/Behaviors/BehaviorsMeshGroup.cpp

```diff
@@ -30,7 +30,7 @@ namespace AZ
     {
         namespace Behaviors
         {
-            const int MeshGroup::s_meshGroupPreferredTabOrder = 0;
+            const int MeshGroup::s_meshGroupPreferredTabOrder = 1;
```

Comment:
These constants can use some comments as groups are ordered in relation to other groups in the FBX Scene  Settings panel.

Or better, yet, this order can be tossed Settings can be tossed in a .setreg file where they are all visible for the user. Also since Rapidjson supports comments in json, a comment can be added as well


```json
{
    "O3DE": {
        "SceneGroup": {
           // Represents the order of tabs in the Export Scene Settings panel
           // Values should be different from each other
            "TabOrder": {
                "ImportGroup": 0
                "MeshGroup": 1,
                "SkeletonGroup": 2,
                "SkinGroup": 3,
                "AnimationGroup": 4
            }
         }
    }
}
```

The Settings Registry can then just be quiered for the value
```c++
constexpr AZStd::string_view SceneGroupTabOrderRootKey =  "/O3DE/SceneGroup/TabOrder";
// ...

// Default mesh group tab order to 1 in case there are not settings for it in the Settings Registry
AZ::s64 meshGroupPreferredTabOrder = 1;
if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
{
    using FixedValueString = AZ::SettingsRegistry::FixedValueString;
    settingsRegistry->Get(meshGroupPreferredTabOrder, FixedValueString(SceneGroupTabOrderKey) + "/MeshGroup");
}
```

Reply:
While I dislike the current hard-coded values, I'm also not keen on the setreg solution because some of these values come from groups in gems, which might not all exist in the core engine.

Actually, I'll also revert the other number changes and just make ImportGroup an extremely low negative value. I'm not trying to affect the location of the other tabs, I'm just trying to make sure Import appears first.

Reply:
I backed out my other tab order changes (and also discovered that I had initially missed a couple in the EMFX gem), added comments to the new tab order constant in ImportGroup, and fixed a bug where negative values didn't work when compared against max int (the default tab order value).

Reply:
That is fine with me.